configs/webpack/webpack.config.renderer.dev.dll.ts

```
@@ -30,7 +30,7 @@ const configuration: webpack.Configuration = {
   /**
    * Use `module` from `webpack.config.renderer.dev.js`
    */
-  // eslint-disable-next-line @typescript-eslint/no-var-requires
+   
   module: require('./webpack.config.renderer.dev').default.module,
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)